.travis.yml

```diff
@@ -44,9 +44,20 @@ jobs:
   - env: SQLFLOW_TEST_DB=mysql
     script:
     - set -e
+    # Build MySQL server image including datasets.
+    - cd $TRAVIS_BUILD_DIR
+    - docker build -t sqlflow:mysql -f docker/mysql/Dockerfile .
```

Comment:
We may not need to build the MySQL + datasets image every time, it seems never changes. If the MySQL Docker image have changed, we can manually push it.

Reply:
Agree with your, and maybe we can also publish `sqlflow/sqlflow:mysql` on CI. And `docker pull sqlflow/sqlflow:mysql` to cache the build?

Reply:
No, `sqlflow/sqlflow:mysql` almost never changes, just pull the image and use to save some CI time.